main.js

```diff
@@ -24,35 +53,38 @@ let todoComponent = div(
         () => [todos]
     ),
     dynamic(
-        () => list(todos.map((todo) => p(todo))),
+        () => list(todos.map((todo) => div(link(todo, "/todo/" + todo + "/info?param1=test&hello=world")))),
         () => [todos]
     )
 );
 
-let rootDiv = div(
-    h1("Hoi!"),
-    h2("Hoi2!"),
+// setup router
+useRouter().on("/", mainPage);
 
-    span("Red text here :)").color("red"),
-
-    p("This is a paragraph.").on("dblclick", () => alert("Hello Universe")),
-
-    dynamic(
-        () => p(`Counter: ${counter} `, counter % 2 === 0 && "is Even"),
-        () => [counter]
-    ),
+// access router
+router()
```

Comment:
Is it also possible to register all routes with useRouter? What's the difference there?

Reply:
It's the same. There is no difference. `useRouter` returns the reference of the `Router` object. The only difference is that you can enable and disable the router with `useRouter(true|false)`. When the router is disabled `router()` returns null.